packages/editor/src/components/convert-to-group-buttons/convert-button.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+import { MenuItem } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { createBlock } from '@wordpress/blocks';
+import { withSelect, withDispatch } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+
+export function ConvertToGroupButton( {
+	onConvertToGroup,
+} ) {
+	return (
+		<Fragment>
+			<MenuItem
+				className="editor-block-settings-menu__control block-editor-block-settings-menu__control"
+				icon="controls-repeat"
+				onClick={ onConvertToGroup }
+			>
+				{ __( 'Convert to Group' ) }
```

Comment:
Maybe just "Group" here (adding a keyboard shortcut would be nice).

Reply:
Done

Reply:
@mtias Do you have an example anywhere of where I could add a keyboard shortcut that applies to the editor canvas? I have found `packages/components/src/keyboard-shortcuts/` but not sure on best practise here.

Reply:
This component should be a good start https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/block-editor-keyboard-shortcuts/index.js

Reply:
Thanks. I've implemented this.